Bug: test_vlan_ping fails on dualtor-120 with AssertionError: Did not receive expected packet on any of ports [] for device 0.

### Is it platform specific

generic

### Importance or Severity

Critical

### Description of the bug

 vlan/test_vlan_ping.py selects Port-Channel2 for the lower ToR only when the
  topology name contains "dualtor-aa".

  On dualtor-120 lower-ToR runs, the test still uses Port-Channel1 VM data while
  matching against the lower-ToR minigraph. The minigraph lookup does not find the
  expected uplink ports, leaving vmhost_info["port_index_list"] empty.

  The test then fails packet verification because it expects the packet on an
  empty port list.


### Steps to Reproduce

1. Use a dualtor-120 testbed.
2. Select the lower ToR as the DUT under test.
3. Run:

`     pytest vlan/test_vlan_ping.py::test_vlan_ping`

  4. Observe that vlan_ping_setup builds vmhost_info with an empty
     port_index_list.


### Actual Behavior and Expected Behavior

 **Actual behavior:**

  The test fails with packet verification against an empty expected port list:

  AssertionError: Did not receive expected packet on any of ports [] for device 0.

  or:

  AssertionError: Received expected packet on port <port> for device 0, but it
  should have arrived on one of these ports: [].

  **Expected behavior**:

  For any dualtor topology, when the selected DUT is the lower ToR, the setup
  should use the lower-ToR Port-Channel2 VM path. vmhost_info["port_index_list"]
  should contain the expected PTF port indices, and packet verification should
  use those ports.



### Relevant log output

```shell
vlan_ping_setup = ({
    'ipv4': IPv4Address('10.0.0.57'),
    'ipv6': IPv6Address('fc00::72'),
    'mac': '0e:09:4a:09:53:74',
    'port_index_list': []
  }, ...)

  tbinfo = {'conf-name': 'dt120', ...}
  rand_one_dut_hostname = 'gd272'
  duthost = <MultiAsicSonicHost gd272>

  AssertionError: Did not receive expected packet on any of ports [] for device 0.
```

### Output of `show version`

```shell

```

### Attach files (if any)

_No response_

--- a/tests/vlan/test_vlan_ping.py
+++ b/tests/vlan/test_vlan_ping.py
@@ -91,7 +91,7 @@ def vlan_ping_setup(duthosts, rand_one_dut_hostname, ptfhost, nbrhosts, tbinfo, 
             interface_name = 'Ethernet1'
             dev_name = 'eth1'
         # in case of lower tor host we need to use the next portchannel
-        if "dualtor-aa" in tbinfo["topo"]["name"] and rand_one_dut_hostname == lower_tor_host.hostname:
+        if "dualtor" in tbinfo["topo"]["name"] and rand_one_dut_hostname == lower_tor_host.hostname:
             interface_name = 'Port-Channel2'
             dev_name = _portchannel_netdev_name(vm_info['host'], 2)
 

